CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## kube-ps1 CHANGELOG
+
+## UNRELEASED
+
+* Color function cleanup
+* `_kube_ps1_binary()` check moved to `_kube_ps1_update_cache()`
```

Comment:
People don't care about these two at all.
Changelog is for behavioral changes that your users see.

Reply:
Ok, thank you.